src/Sulu/Bundle/AdminBundle/Resources/js/components/ItemSelection/ItemSelection.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+import React from 'react';
+import type {ChildrenArray, Element} from 'react';
+import {SortableContainer, SortableElement, SortableHandle, arrayMove} from 'react-sortable-hoc';
+import type {Button} from './types';
+import Header from './Header';
+import Item from './Item';
+import itemSelectionStyles from './itemSelection.scss';
+
+type Props = {
+    children: ChildrenArray<*>,
+    /** The text inside the header bar of the `ItemSelection` */
+    label?: string,
+    /** Called when the remove button is clicked on an item */
+    onItemRemove?: (itemid: string | number) => void,
+    /** Called after a drag and drop action was executed */
+    onItemsSorted?: (oldIndex: number, newIndex: number) => void,
+    /** The config of the button placed left side of the header */
+    leftButton?: Button,
+    /** The config of the button placed right side of the header */
+    rightButton?: Button,
+};
+
+export default class ItemSelection extends React.PureComponent<Props> {
+    static Item = Item;
+
+    static arrayMove = arrayMove;
```

Comment:
I am not sure if we also want to export functions like that.

Reply:
Talked to @chirimoya, we will remove that and add this as export to `containers/index.js`, so that we could change it later, in case we ditch `react-sortable-hoc`.

Reply:
Sorry, that should be `components/index.js`.